README.md

```diff
@@ -1,11 +1,11 @@
-# Deferat
-
+## 🚀 Deferat
 A simple blog engine for displaying markdown formatted posts written in .NET Core.
-
 ![](https://github.com/euronay/deferat/workflows/Build/badge.svg)
 
 ### Usage
 
+Simply run:
+## 📘 Usage
```

Comment:
You've got duplicate "Simply Run" and "Usage" lines, could you tidy these up, please?